ajax/policyValue.php

```diff
@@ -42,9 +42,23 @@
 }
 
 $policyId = intval($_POST['value']);
```

Comment:
While you're here, can you replace intval(...) by a (int) cast ? (for performance, and (int) is sufficient)